docker/conda/environments/cuda11.8_dev.yml

```diff
@@ -31,11 +31,12 @@ dependencies:
     - click >=8
     - cmake=3.24
     - configargparse=1.5
+    - cuda-compiler=11.8
     - cuda-nvml-dev=11.8
```

Comment:
If the channel `nvidia/label/cuda-11.8.0` is before `nvidia`, can we remove the version from all of the `cuda-*` packages? That way we can reduce the number of places that you specify the cuda version.

Reply:
possibly. I'll check now that it's in place.

Reply:
no, because what we really want to do is move conda-forge before any nvidia channels to better fit with the rapids ecosystem.